### ISSUE TITLE
Color codes for 1.11.2 Leave Artifacts In Log

### ISSUE THREAD

**Issue by TwitchtvCarbs** [GenDeathrow/MPUtils#12]
This has actually been an issue since I installed but I never got around to reporting it D:

When you add color coding to your change log you get the color but you also get an extra couple characters in front of the line. Shown below:

![mputilsbug](https://cloud.githubusercontent.com/assets/25915492/25632784/337af990-2f32-11e7-9ae9-77ffc151462b.JPG)

**Comment by GenDeathrow** [comment]
Yeah it's been an issue everytime I think I get it. It comes back. Normally a quick fix is create a new text file and copy the text. Or use the in-game text editor when Edit Mode is on.  But 1.11 I may just need to test more in that version. Are you using the hex codes? Normally that's better for text files like this.

**Comment by TwitchtvCarbs** [comment]
Oh I'm not I was just using the symbol and single number/letter. Would it just be like #HexCode before the text?

**Comment by GenDeathrow** [comment]
Sorry I said hex.. I mean unicode I'll have to find an example.
